lib/reporter-proxy.js

```diff
@@ -4,22 +4,24 @@ export default class ReporterProxy {
   constructor () {
     this.reporter = null
     this.queue = []
+    this.eventType = 'welcome-v1'
   }
 
   setReporter (reporter) {
     this.reporter = reporter
     let customEvent
+
     while ((customEvent = this.queue.shift())) {
-      this.reporter.addCustomEvent(customEvent.category, customEvent)
+      this.reporter.addCustomEvent(this.eventType, customEvent)
```

Comment:
so the reason we were passing `category` as `eventType` was to be consistent with what we were passing to google analytics.  Now that google analytics doesn't exist any more, consistency isn't important there.   This may break some of our existing dashboards, if the event type differs from the category, so that's something to be aware of.

Reply:
The interesting part is that the `welcome` package was not passing a category before this PR (well it was passing a `category` field, but this is not understood by our pipelines, which expects `ec`), so I don't expect things to get broken (or at least more broken than currently 😝).